app.py

```diff
@@ -12,25 +12,29 @@
 
 @app.route('/', methods=['post', 'get'])
 def login():
-	location = ''
 	if request.method == 'POST':
 		location = request.form.get('address/location')
-		img_bytes = get_image(location)
-		image = add_bernie(img_bytes)
+		if location.strip() != '':
+			img_bytes = get_image(location)
+			if not img_bytes:
+				return render_template('index.html', image=False, location = False)
 
-	if not location:
-		return render_template('index.html', image=False, location = False)
+			image = add_bernie(img_bytes)
+		else:
+			return render_template('index.html', image=False, location = False)
 
-	with open('img.jpeg', 'wb+') as f:
-		f.write(image.read())
-		image.seek(0)
+		with open('img.jpeg', 'wb+') as f:
+			f.write(image.read())
+			image.seek(0)
 
-	print(type(image))
-	img_url = base64.b64encode(image.getvalue())
+		print(type(image))
+		img_url = base64.b64encode(image.getvalue())
 
-	# print(img_url)
+		# print(img_url)
+		return render_template('index.html', image=img_url.decode('utf-8'), location=location)
 
-	return render_template('index.html', image=img_url.decode('utf-8'), location=location)
+	#GET Request Response
+	return render_template('index.html', image=False, location = False)
```

Comment:
Would it make more sense to just return a simple line asking to use "POST" request instead? I am asking coz when the user enters location in this new Get page and hits submit, it still returns "Internal Server Error" which is undesirable.